configure.ac

```diff
@@ -209,9 +209,14 @@ AC_DEFUN([CONFIG_OPTION_MYSQL],[
 
   MYSQL_CXXFLAGS="-felide-constructors"
   case "$MYSQL_MAJOR_MINOR_VERSION" in
-  5.5|5.6|10.*)
+  5.5|5.6|10.[0-3])
     MYSQL_CXXFLAGS="$MYSQL_CXXFLAGS -fno-rtti"
     ;;
+  10.4)
```

Comment:
Could you put this before `5.5|5.6|...)`?

Reply:
I've fixed.